source/index.html.md.erb

```diff
@@ -10945,14 +11273,6 @@ Creates a copy of the specified Sheet.
 
 ## Create Sheet
 
-To create a Sheet from scratch, use one of the following operations (according to where you want to create the Sheet):
```

Comment:
I think this is helpful, though could be tightened up

Reply:
Done